code/modules/mob/living/living.dm

```diff
@@ -742,7 +742,7 @@ Thanks.
 								if (bleeding_organs.len)
 									var/blood_volume = round(HM:vessel.get_reagent_amount("blood"))
 									/*Sometimes species with NO_BLOOD get blood, hence weird check*/
-									if(blood_volume > 0 || (HM.species.anatomy_flags & NO_BLOOD))
+									if(blood_volume > 0 || !(HM.species.anatomy_flags & NO_BLOOD))
```

Comment:
So, if they have blood, or if they don't have the NO_BLOOD flag?

This is just making less and less sense. Move the NO_BLOOD check up to if (bleeding_organs.len) So it checks if they have any open wounds and if they don't have the NO_BLOOD flag